src/amo/components/AddonReview.js

```diff
@@ -104,15 +94,6 @@ export class AddonReviewBase extends React.Component {
         <form onSubmit={this.onSubmit} ref={(ref) => { this.reviewForm = ref; }}>
           <div className="AddonReview-form-input">
             {errorHandler.hasError() ? errorHandler.renderError() : null}
-            <input
```

Comment:
Did this never have a label? Looks like the `<textarea>` is missing one too. I know it's unrelated to this patch, but could you add one?